vllm/core/block/cpu_gpu_block_allocator.py

```diff
@@ -92,11 +88,6 @@ def create(
 
     def __init__(self, cpu_block_allocator: BlockAllocator,
                  gpu_block_allocator: BlockAllocator):
-        assert not (
-            cpu_block_allocator.all_block_ids
-            & gpu_block_allocator.all_block_ids
-        ), "cpu and gpu block allocators can't have intersection of block ids"
```

Comment:
let's keep this assert in

Reply:
fixed in [04cadfd](https://github.com/vllm-project/vllm/pull/5968/commits/04cadfdb0f825f59825fdaf884fd1bf2e0e783b6)